src/main.cpp

```diff
@@ -46,6 +46,10 @@ extern "C" {
 #endif
 }
 
+#if TARGET_OS_IPHONE
+#include <SDL2/SDL.h>
+#endif
```

Comment:
unused?

Reply:
Not, this replaces main with sdl_main, SDLmain.a does some initialisation and calls sdl_main...